org.opentosca.bus.management.deployment.plugin.remote/src/org/opentosca/bus/management/deployment/plugin/remote/ManagementBusDeploymentPluginRemote.java

```diff
@@ -9,7 +9,7 @@
 import org.apache.camel.Message;
 import org.opentosca.bus.management.deployment.plugin.IManagementBusDeploymentPluginService;
 import org.opentosca.bus.management.header.MBHeader;
-import org.opentosca.bus.management.service.impl.collaboration.Constants;
+import org.opentosca.bus.management.service.impl.Constants;
```

Comment:
Was this essential to the change? But LGTM :)

Reply:
By moving the plan instance handling to the bus, we need some constants not related to the collaboration between different containers. Therefore, we either need two constants classes or move the existing one from the collaboration to the general package and I think thats the better solution :-)

Reply:
I personally dislike the idea of a `Constants` class for various reasons, the primary one being that names should be scoped to the smallest applicable scope. There is precedent for multiple `Constants` classes, though. Examples include `MBHeaders`, vast majorities of the Settings, which are functionally constant, as well as classes used for Routing configuration within the main bus.